### ISSUE TITLE
Installation stuck after installing fixtures of prestashop-shop-creator

### ISSUE THREAD

**Issue by tom-combet** [PrestaShop/PrestaShop#28726]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

After using generated data from [prestashop-shop-creator](https://github.com/PrestaShop/prestashop-shop-creator) with default configuration, installation is blocked after installFixtures step.

![image](https://user-images.githubusercontent.com/48719560/173069167-4480d2db-5883-4d83-8a53-625efc54b2fd.png)

![image](https://user-images.githubusercontent.com/48719560/173069641-0038c369-0c94-4eee-b245-e2d32523fd38.png)

### Expected behavior

Installation should display that it's finished.

### Steps to reproduce

1. `git clone https://github.com/PrestaShop/prestashop-shop-creator.git`
2. `cd prestashop-shop-creator`
3. Under Vendor/fakerphp/src/Facker/Provider/Image.php, add the line below after this line `curl_setopt($ch, CURLOPT_FILE, $fp);`
4. `curl_setopt($ch, CURLOPT_USERAGENT, 'Mozilla/5.0 (Windows NT 6.2; WOW64; rv:17.0) Gecko/20100101 Firefox/17.0');`
5. Execute `composer install`
6. Setup the number of data you wish for each categories 
7. `php app/console.php`
8. Begin installation of PrestaShop and stop at database configuration step
9. Move generated data to install/fixtures/fashion folder of PrestaShop installation
10. Under fashion/data, replace `dni=""` by `dni="12345678A"` in `address.xml` generated file.
11. Continue installation
12. See error > the installation is done, yet the interface still displaying "in progress" with "101%"

### PrestaShop version(s) where the bug happened

1.7.8.6

### PHP version(s) where the bug happened

7.4

**Comment by florine2623** [comment]
Hello @tom-combet ,

Could you please provide a screen record, because the step 3 is a little confusing ?

Thanks!

**Comment by tom-combet** [comment]
Did you successfully generated the data with prestashop-shop-creator?
If so, you just have to move the `generated_data` folder content to `install/fixtures/fashion` folder, in your new PrestaShop you're installing, like so:

`mv /path/to/prestashop-shop-creator/generated_data/* /path/to/prestashop/install/fixtures/fashion`

**Comment by AureRita** [comment]
Hello @tom-combet, 

Please check if these steps to reproduce are correct : 

- create a folder where you want to put the files generated
- git clone https://github.com/PrestaShop/prestashop-shop-creator.git
- composer install
- setup the number of data you wish for each categories 
- php app/console.php
- Begin installation of PrestaShop and stop at database configuration step
- Move generated data to install/fixtures/fashion folder of PrestaShop installation
- Continue installation

Waiting for your feedback 

Thanks

**Comment by tom-combet** [comment]
> Hello @tom-combet,
> 
> Please check if these steps to reproduce are correct :
> 
>     * create a folder where you want to put the files generated
> 
>     * git clone https://github.com/PrestaShop/prestashop-shop-creator.git
> 
>     * composer install
> 
>     * setup the number of data you wish for each categories
> 
>     * php app/console.php
> 
>     * Begin installation of PrestaShop and stop at database configuration step
> 
>     * Move generated data to install/fixtures/fashion folder of PrestaShop installation
> 
>     * Continue installation
> 
> 
> Waiting for your feedback
> 
> Thanks

Yup, except that step 1 is not compulsory: prestashop-shop-creator will already create a folder named "generated_data" for you.

**Comment by hibatallahAouadni** [comment]
Hello @tom-combet 

I encountered another issue while installation, see the attached screenshot below:

![image](https://user-images.githubusercontent.com/23308427/174040633-51e02e94-70d1-4415-8962-e739bd962032.png)

Did you encounter the same issue?
Waiting for your feedback.

Thanks!

**Comment by tom-combet** [comment]
Ah yep, I had the problem, but it doesn't come from prestashop-shop-creator: it comes from [placeholder.com](https://github.com/FakerPHP/Faker/issues/475) by Faker.
I solved it with the [temporary solution](https://github.com/FakerPHP/Faker/issues/475#issuecomment-1124912209) given in this very issue.

**Comment by hibatallahAouadni** [comment]
Hello @tom-combet 

Thanks for your solution, it helps :tada: 
But, I encounter another issue :sweat_smile: see the attached screenshot below:

![image](https://user-images.githubusercontent.com/23308427/174260127-a01e6603-8681-4f2e-aff8-74b4f8aca267.png)

Please help :sweat: waiting for your feedback.

Thanks!

**Comment by tom-combet** [comment]
Ah yep, known issue: https://github.com/PrestaShop/prestashop-shop-creator/issues/8

Same thing, temporary solution in the issue discussion.

**Comment by hibatallahAouadni** [comment]
Hello @tom-combet 

Thanks for your help and feedbacks, I reproduce the issue with PS1786, see the attached screne record below:

https://www.awesomescreenshot.com/video/9541621?key=cc49786d0d0d9950284b2e63d38a3cd8

The first bug: it shouldn't display 101%
The second bug: the installation interface was blocked even though the process completed successfully
I’ll add this to the debug backlog so that it’s fixed.

Please be aware that due to the high number of reported issues, some bugs might take a very long time to be resolved. If this issue is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you. 

If you manage to get the problem fixed, then please contribute the fix back to the project as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.

Thank you

**Comment by Progi1984** [comment]
@AureRita & @tom-combet Images & DNI errors are fixed in `prestashop-shop-creator`
